updater/main.go

```diff
@@ -158,13 +165,21 @@ func _main() error {
 		ok, err := u.verifyUpdate(i)
 		if err != nil {
 			log.Printf("Failed to verify update for instance %#q: %v", i, err)
+			summary[i.instanceID] = fmt.Sprintf("Failed to verify update: %v", err)
 		}
 		if !ok {
 			log.Printf("Update failed for instance %#q", i)
+			summary[i.instanceID] = "Update failed"
 		} else {
 			log.Printf("Instance %#q updated successfully!", i)
+			summary[i.instanceID] = "Instance updated successfully"
```

Comment:
I think we should only have one line of summary for each instance. All the conditions here are not mutually exclusive and would end up replacing other. I think below should be okay in summary.
```suggestion
			
		}
		if !ok {
			log.Printf("Update failed for instance %#q", i)
			summary[i.instanceID] = "Update failed"
		} else {
			log.Printf("Instance %#q updated successfully!", i)
			summary[i.instanceID] = "Instance updated successfully"
```